src/modules/add.js

```diff
@@ -1,8 +1,4 @@
-import deleteTask, {
-  clearfield, tasks, checkTasks, clearAll,
-} from './delete.js';
-
-// import checkTasks from './interactive.js';
+import deleteTask, { clearfield, tasks } from './delete.js';
```

Comment:
- [ ] Kindly add the edit functionality as requested by previous reviewers and as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20editing%20task%20descriptions.).  You will find a demo on how you can do it on this [comment](https://github.com/Nezerwa/Todolist-app/pull/2#pullrequestreview-1264669799).